README.md

```diff
@@ -53,3 +53,26 @@ openshift-install destroy cluster
 Note that you almost certainly also want to clean up the installer state files too, including `auth/`, `terraform.tfstate`, etc.
 The best thing to do is always pass the `--dir` argument to `install` and `destroy`.
 And if you want to reinstall from scratch, `rm -rf` the asset directory beforehand.
+
+
+### Troubleshouting installation
+
+```Cleanup``` step should clean everything what was created during previous installation. However, sometimes installer can report error messages as ones below 
```

Comment:
```Cleanup``` step *should* tear down everything created during the previous installation, however sometimes the installer may report error messages like the ones below: